tests/Unit/PointwiseFunctions/AnalyticSolutions/Poisson/Test_Moustache.cpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cmath>
+#include <string>
+#include <tuple>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Poisson/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/Poisson/Moustache.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+template <size_t Dim>
+void test_solution() {
+  const Poisson::Solutions::Moustache<Dim> solution{};
```

Comment:
This is to check that the analytic formula coded up in python gives the same numerical results as the analytic formula coded up in SpECTRE right? Is there a test comparing the analytic SpECTRE solution and the numerical solution found by SpECTRE? (if that's possible yet)

Reply:
That's right. There is no test for the numeric solution yet since the elliptic solver is not in spectre yet. Eventually there will be test executables that solve the Poisson equation for some analytic solutions and make sure the solver works.